apps/workspace-host/src/interface.js

```diff
@@ -166,11 +182,16 @@ async
     async () => {
       // Set up a periodic pruning of running containers
       async function pruneContainersTimeout() {
-        await pruneStoppedContainers();
-        await pruneRunawayContainers();
-        await sqldb.queryAsync(sql.update_load_count, {
-          instance_id: workspace_server_settings.instance_id,
-        });
+        try {
+          await pruneStoppedContainers();
+          await pruneRunawayContainers();
+          await sqldb.queryAsync(sql.update_load_count, {
+            instance_id: workspace_server_settings.instance_id,
+          });
+        } catch (err) {
+          logger.error('Error pruning containers', err);
+          Sentry.captureException(err);
+        }
```

Comment:
A small thing I noticed while working on this: previously, if any of these functions threw, the pruning process would essentially die because we'd never schedule another one via the `setTimeout` call below this.